openstack/sfs_turbo/v1/shares/requests.go

```diff
@@ -84,6 +88,52 @@ func List(c *golangsdk.ServiceClient) ([]Turbo, error) {
 	return ExtractTurbos(pages)
 }
 
+// ListOptsBuilder allows extensions to add additional parameters to the List
+// request.
+type ListOptsBuilder interface {
+	ToVolumeListQuery() (string, error)
+}
+
+// ListOpts holds options for listing Volumes. It is passed to the volumes.List
+// function.
+type ListOpts struct {
+	// Requests a page size of items.
+	Limit int `q:"limit"`
+	// Used in conjunction with limit to return a slice of items.
+	Offset int `q:"offset"`
+}
+
+// ToVolumeListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToVolumeListQuery() (string, error) {
+	q, err := golangsdk.BuildQueryString(opts)
+	return q.String(), err
+}
+
+// ListPage returns one page limited by the conditions provided in Opts.
+func ListPage(client *golangsdk.ServiceClient, opts ListOptsBuilder) (*PagedList, error) {
+	url := listURL(client)
+	if opts != nil {
+		query, err := opts.ToVolumeListQuery()
+		if err != nil {
+			return nil, err
+		}
+		url += query
+	}
+
+	var rst golangsdk.Result
+	_, err := client.Get(url, &rst.Body, &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	var r PagedList
+	err = rst.ExtractInto(&r)
+	return &r, err
```

Comment:
if r.Count is not used, can we just return r.Shares?

Reply:
if r, the first return value will be ignored.
if !r,Count and Shares are both needed